pom.xml

```diff
@@ -46,8 +46,7 @@
     </scm>
 
     <properties>
-        <jenkins.version>1.609.3</jenkins.version>
-        <workflow.version>1.14</workflow.version>
+        <jenkins.version>1.642.3</jenkins.version>
```

Comment:
What was producing the linkage error? And why it needs to bump baseline to 1.642.3?